app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -316,12 +317,13 @@ fun questTypeRegistry(
 
     /* ↓ 2.solvable when right in front of it but takes longer to input --------------------- */
 
-    // bike parking: would be higher up if not for bike parking capacity which is usually not solvable when moving past
+    // bike parking/rental: would be higher up if not for bike parking/rental capacity which is usually not solvable when moving past
     AddBikeParkingCover(), // used by OsmAnd in the object description
     AddBikeParkingType(), // used by OsmAnd
     AddBikeParkingAccess(),
     AddBikeParkingFee(),
     AddBikeParkingCapacity(), // used by cycle map layer on osm.org, OsmAnd
+    AddBikeRentalCapacity(),
```

Comment:
Less ambiguous (for docked at least), so less thinking time than normal parking.
```suggestion
    AddBikeRentalCapacity(),
    AddBikeParkingCapacity(), // used by cycle map layer on osm.org, OsmAnd
```

Reply:
```suggestion
    AddBikeRentalCapacity(), // less ambiguous than bike parking
    AddBikeParkingCapacity(), // used by cycle map layer on osm.org, OsmAnd
```